projects/canopy/src/lib/filter-buttons/filter-group.component.html

```diff
@@ -0,0 +1,7 @@
+<fieldset [attr.aria-describedby]="ariaDescribedBy">
```

Comment:
I've noticed that in storybook the code is appearing as `<fieldset aria-describerdby>`. If `ariaDescribedBy` is undefined I believe is better to hide the attribute as this might confuse the screen readers.

 I would simply do something like: `<fieldset [attr.aria-describedby]="ariaDescribedBy || null">`

Reply:
Done.